templates/mks/party_detail.html

```diff
@@ -32,12 +33,12 @@
 {% block description %}{{ object.name }} - {% trans 'Open Knesset - Opening the Knesset to the public' %}{% endblock %}
```

Comment:
@yotammanor For template readability sake: can generic "object" be named for what it really is? I'm guessing party, but just from reading the code it's pure guessing. changing to explicit naming would help other contributors understand what this template is about